apps/client/src/app/pages/resources/resources-page-routing.module.ts

```diff
@@ -7,18 +7,40 @@ import { ResourcesPageComponent } from './resources-page.component';
 
 const routes: Routes = [
   {
```

Comment:
Can you please order the attributes alphabetically?

Reply:
Hey sorry about that I missed those ones. Have handled that as well.